app/templates/search.html

```diff
@@ -1,14 +1,39 @@
 {% extends "base.html" %}
 {% import '_snippets.html' as snippets %}
+{% block scripts %}
+  <script>
```

Comment:
@rufuspollock need javascript to put search term in search box. As we discussed search-box should contain search term.